src/modules/modules_common.c

```diff
@@ -10,7 +10,7 @@
 
 #include <logging/log.h>
 
-LOG_MODULE_REGISTER(modules_common, CONFIG_CAT_TRACKER_LOG_LEVEL);
+LOG_MODULE_REGISTER(modules_common, CONFIG_APPLICATION_MODULE_LOG_LEVEL);
```

Comment:
I wonder if we should define an own logging instance for the ```modules_common``` library and strictly reserve ```CONFIG_XXXXX_MODULE_LOG_LEVEL``` for the respective module.

Reply:
@jtguggedal @jhn-nordic Are we going for this kind of segmentation when it comes to logging or do we allow our self to reuse log configurable options across source files?

Reply:
I would vote for adding a seperate log level for modules_common

Reply:
Updated the PR to makes sure that log level configs are only used by its respective lib/module